### PR TITLE
[model] feat: support wan2.2 i2v sft

### PR DESCRIPTION
**Pull request by yuyu5333**
# Add model Wan2.2 I2V SFT

- support Wan2.2 I2V sft

## Key Changes
- Wan2.2 MOE independent training
- train.max_timestep_boundary
- train.min_timestep_boundary

## Wan2.2-I2V-A14B SFT

Fine-tuning the Wan2.2-I2V-A14B model requires fine-tuning high_noise and low_noise separately. By controlling different timestep_boundaries, the performance of the model at different stages can be controlled. The SFT training script is as follows：

- high_noise:
```
bash train.sh tasks/omni/train_wan2_2.py configs/dit/wan2_2_sft.yaml --model.model_path Wan-AI/Wan2.2-I2V-A14B/high_noise_model --train.max_timestep_boundary 1.0 --train.min_timestep_boundary 0.875
```

- low_noise:
```
bash train.sh tasks/omni/train_wan2_2.py configs/dit/wan2_2_sft.yaml --model.model_path Wan-AI/Wan2.2-I2V-A14B/low_noise_model --train.max_timestep_boundary 0.875 --train.min_timestep_boundary 0.0
```

# TODO
- [x] Train LoRa 
- [ ] DataProcess
- [ ] Flashattention3 at veomni/ops/attention.py

### CODE REVIEW

**Comment by yuyu5333** [comment]
Support Wan2.2 I2V training submitted in a new PR ([[model] feat: support wan2.2 i2v #45](https://github.com/ByteDance-Seed/VeOmni/pull/45))
